server/context.jsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from "react";
+
+const PageFeatures = createContext(null);
```

Comment:
There is no need to separate the context from `_app.jsx` if you don't create a custom hook out of it or use more complex logic.
This should be in `_app.jsx` at the top.

Reply:
noted